File: Back End/Entity/Models/AppUser.cs
using Core.Interfaces;
using Entity.Models;
using Microsoft.AspNetCore.Http;
using Microsoft.AspNetCore.Identity;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Entity.Models
{
    public class AppUser:IdentityUser,IEntity
    {
        public int CamexId { get; set; }
        public Balance Balance { get; set; }
        public int CityId { get; set; }
        public virtual City City { get; set; }
        public string Address { get; set; }
        public string ActiveCode { get; set; }
        public int OfficeId { get; set; }
        public virtual Office Office { get; set; }
        public string Image { get; set; }
        [NotMapped]
        public IFormFile Photo { get; set; }
        public ICollection<Cargo> Cargos { get; set; }
        public ICollection<Order> Orders { get; set; }
        public ICollection<Receipt> Receipts { get; set; }
        public bool IsTermsAccepted { get; set; }
        public bool IsActived { get; set; }
        public bool IsDeleted { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime ModifiedDate { get; set; }

        public AppUser()
        {
            this.CreatedDate = DateTime.UtcNow;
            this.ModifiedDate = DateTime.UtcNow;
        }

    }
}

File: Front End/src/app/Admin/Models/AppUser.ts
import {Balance} from './Balance';
import {Office} from '../../Core/models/Office';
import {City} from '../../Core/models/City';
import {Cargo} from '../../Core/models/Cargo';
import {Order} from '../../Core/models/Order';
import {Receipt} from '../../Core/models/Receipt';

export class AppUser{
  id: string;
  email: string
  phoneNumber: string
  camexId: number;
  balance: Balance;
  cityId: number;
  city: City;
  address: string;
  officeId: number;
  office: Office;
  image: string;
  cargos: Cargo[];
  orders: Order[];
  receipts: Receipt[];
  createdDate: Date;
  modifiedDate: Date;


}
